### PR TITLE
Update Docs: migrate Beta Features - GraphQL APIs

### PR DESCRIPTION
**Pull request by privatemaker**
Continuing migration of Beta Features Documentation #7040

@martinjagodic i'll stop here until I get a confirmation my workflow here works for you 😉

### CODE REVIEW

File: website/content/docs/github-backend.md [hunk 1]
```diff
@@ -39,3 +39,23 @@ The above configuration would look for the status who's `"context"` is `"my-prov
 ## Git Large File Storage (LFS)
 
 Please note that the GitHub backend **does not** support [git-lfs](https://git-lfs.github.com/), see [this issue](https://github.com/decaporg/decap-cms/issues/1206) for more information.
+
+
+## GraphQL API
+
+Experimental support for GitHub's [GraphQL API](https://developer.github.com/v4/) is now available for the GitHub backend.
```
Comment (martinjagodic) [inline website/content/docs/github-backend.md:46]: Let's remove this line as it's not flagged as experimental anymore.

File: website/content/docs/gitlab-backend.md [hunk 2]
```diff
@@ -58,6 +58,32 @@ With GitLab's PKCE authorization, users can authenticate with GitLab directly fr
      auth_endpoint: oauth/authorize
    ```
 
+
+**Note:** In all cases, GitLab also provides you with a client secret. You should *never* store this in your repo or reveal it in the client.
+
+
+## GraphQL API
+
+Experimental support for GitLab's [GraphQL API](https://docs.gitlab.com/ee/api/graphql/) is now available for the GitLab backend.
```
Comment (martinjagodic) [inline website/content/docs/gitlab-backend.md:67]: Let's remove this line as it's not flagged as experimental anymore.

**Review by martinjagodic — CHANGES_REQUESTED** [review]

**Comment by privatemaker** [comment]
Pushed your request and moved the **Note: ...** bit to bottom.

**Review by martinjagodic — COMMENTED** [review]
@privatemaker I would keep the note under title. It could be frustrating for someone to go over the tutorial and only then realising it's not compatible with their setup. What do you think?

**Comment by privatemaker** [comment]
Hrm, if a user is interested in using `github` or `gitlab` backend, I assume user knows they are not using `git-gateway` as a backend. So why the need for the **Note: ...** at all?

But not that big of a deal. I moved it back under the header.

**Review by martinjagodic — APPROVED** [review]

**Pull request merged**